doc/spec.md

```diff
@@ -1031,7 +1036,8 @@ fib(5)
 ```
 
 This rule, combined with the invariant that all loops are iterations
-over finite sequences, implies that Starlark programs are not Turing-complete.
+over finite sequences, implies that Starlark programs are not
+Turing-complete when the `-rec` flag is not specified.
```

Comment:
"...programs cannot be Turing complete unless the -recursion flag is specified".